src/routes/+page.svelte

```diff
@@ -186,14 +186,103 @@
             </p>
         </div>
     </div>
-
+    
     <div class="text-center">
         <a class="inline-flex gap-x-4 items-center bg-green hover:bg-green-dark py-4 px-6 rounded-full text-xl font-ibm font-medium" href="https://eip2077.info">Explore EIP-2077 <span class="inline-block"><Explore /></span></a>
     </div>
+    
 </section>
 
-<section>
+<section class="flex flex-col gap-y-16 px-6 lg:px-20 py-16">
+    <div class="flex flex-col gap-y-4">
+        <h2 class="text-6xl font-inter font-medium">
```

Comment:
I see you're repeating `font-inter` a lot of times. To avoid that you can add this to your `app.css`

```css
body {
    font-family: 'Inter', sans-serif;
}
```

And you'll only need to worry about fonts that aren't `font-inter`